tests/yam/yast_modules/control_center.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Start containerized YaST Control Center with ncurses
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi qw(select_console enter_cmd wait_still_screen save_screenshot get_var);
+
+sub run {
+    my $podman_cmd = "podman container runlabel run registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/yast-mgmt-ncurses-test-api";
+    select_console('root-console');
+    enter_cmd(get_var('YUI_PARAMS') . " $podman_cmd");
+    my $control_center = $testapi::distri->get_control_center();
+    $control_center->open_release_notes();
+    my $release_notes = $testapi::distri->get_release_notes();
+    wait_still_screen(2);
```

Comment:
please remove those `wait_still_screen` those are just sleep, for now with save_screenshot is enough and if we don't get the right screen, we can improve on that later. We need test that are easy to read, not full of hacks :)

Just in case is possible, is it possible to check some other control that would ensure to be ready to take the screenshot? the waiting mechanism should be done through the libyui-rest-api not the test api, only use test-api for screnshots, nothing else.